examples/llava/clip.cpp

```diff
@@ -2823,15 +2823,18 @@ void clip_free(clip_ctx * ctx) {
     delete ctx;
 }
 
+// deprecated
 size_t clip_embd_nbytes(const struct clip_ctx * ctx) {
-    return clip_n_patches(ctx) * clip_n_mmproj_embd(ctx) * sizeof(float);
+    const int32_t nx = ctx->vision_model.hparams.image_size;
+    const int32_t ny = ctx->vision_model.hparams.image_size;
+    return clip_embd_nbytes_by_img(ctx, ny, nx);
 }
 
 size_t clip_embd_nbytes_by_img(const struct clip_ctx * ctx, int img_h, int img_w) {
```

Comment:
Rename `img_h` and `img_w` to `nx` and `ny` and fix calls:

```c++
clip_embd_nbytes_by_img(ctx, ny, nx); <-- wrong
clip_embd_nbytes_by_img(ctx, nx, ny);
```

Reply:
Fixed in https://github.com/ggml-org/llama.cpp/pull/13141/commits/496f1ce8e27997370f481d67ff708a24c97b7629